autosklearn/evaluation/util.py

```diff
@@ -6,7 +9,8 @@
 ]
 
 
-def read_queue(queue_):
+def read_queue(queue_: multiprocessing.Queue
+               ) -> List[Union[str, int, float, List, Dict, Tuple]]:
```

Comment:
Shouldn't each list entry be a `Tuple` with some specified types?

Reply:
I think it should be a Dict... or so I see in the code...